### PR TITLE
add update note to ESI demo server setup

### PR DESCRIPTION
**Pull request by DanNiESh**
include setup requirements, steps and previous issues

### CODE REVIEW

File: update.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+sudo -E tripleo-repos -b ussuri current
+```
+* Install the TripleO CLI
+```
+sudo yum install -y python3-tripleoclient
+```
+* git clone esi-sandbox repo and cd esi-sandbox
+* Open deploy.sh, change INTERFACE=eth0 to INTERFACE=ens10; comment out ceph-ansible related commands.
+* Open standalone_parameters.yaml
+    * remove resource_registry: DockerRegistry, MistralEngine, MistralApi, MistralExecutor, MistralEventEngine
```
Comment (tzumainn) [inline update.md]: I would just remove these from standalone_parameters.yaml and not even mention them here.

File: update.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Enable the current Ussuri repositories
+```
+sudo -E tripleo-repos -b ussuri current
+```
+* Install the TripleO CLI
+```
+sudo yum install -y python3-tripleoclient
+```
+* git clone esi-sandbox repo and cd esi-sandbox
+* Open deploy.sh, change INTERFACE=eth0 to INTERFACE=ens10; comment out ceph-ansible related commands.
```
Comment (tzumainn) [inline update.md]: This is going to be environment dependent; maybe just a note that a user may have to change INTERFACE in deploy.sh?

File: update.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Open standalone_parameters.yaml
+    * remove resource_registry: DockerRegistry, MistralEngine, MistralApi, MistralExecutor, MistralEventEngine
+    * change ContainerImagePrepare namespace to docker.io/tripleou
```
Comment (tzumainn) [inline update.md]: I think I'd just update this in the actual file as well.

File: update.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    * change NeutronPublicInterface value to ens10
```
Comment (tzumainn) [inline update.md]: Environment-dependent.

File: update.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+sudo rm -rf \
+    /var/lib/tripleo-config \
+    /var/lib/config-data /var/lib/container-config-scripts \
+    /var/lib/container-puppet \
+    /var/lib/heat-config \
+    /var/lib/image-serve \
+    /var/lib/containers \
+    /etc/systemd/system/tripleo* \
+    /var/lib/mysql/*
+sudo systemctl daemon-reload
```
Comment (tzumainn) [inline update.md]: This is an *excellent* thing to include! Can you actually make it a script in the repository, and just mention that script here?

**Review by tzumainn — COMMENTED** [review]
Nice document! A few comments inline. Can you also rename it to something more specific - 'ussuri-install.md' maybe - and link to it from the README?

Just to confirm, this installs Ironic, correct?

**Comment by DanNiESh** [comment]
> Nice document! A few comments inline. Can you also rename it to something more specific - 'ussuri-install.md' maybe - and link to it from the README?
> 
> Just to confirm, this installs Ironic, correct?

Yes. Ironic is listed in OpenStack services after deployment.

File: update.md (hunk 1 above)
Comment (DanNiESh) [reply update.md]: An update: after I removed MistralEngine, MistralApi, MistralExecutor, MistralEventEngine and redeploy, the deployment failed. When I added them back, it succeeded again. I'm not sure if Mistral should be removed in this case.

File: update.md (hunk 2 above)
Comment (DanNiESh) [reply update.md]: Make sense!

File: update.md (hunk 5 above)
Comment (DanNiESh) [reply update.md]: Ok!

File: update.md (hunk 1 above)
Comment (tzumainn) [reply update.md]: Oh, interesting - thanks for testing! What error did you get?

Comment (DanNiESh) [reply update.md]: The error was Container(s) with bad ExitCode: [''haproxy_restart_bundle'']

Comment (tzumainn) [reply update.md]: Okay, that's really weird then. Is it possible that mistral was still referenced somewhere else - maybe in deploy.sh?

Comment (DanNiESh) [reply update.md]: deploy.sh has already removed mistral. I find that patches/docker directory references mistral, but not sure if they are used or not.

Comment (tzumainn) [reply update.md]: Ah, they're not. Maybe see if it can be removed?

Comment (DanNiESh) [reply update.md]: After removing mistral related files in patches/docker, the same error still happened.

**Review by tzumainn — APPROVED** [review]
Great - thanks for the docs!

**Pull request merged**